frontends/election-manager/src/components/smartcard_modal/card_details_view.tsx

```diff
@@ -103,70 +111,72 @@ export function CardDetailsView({
     });
   }
 
-  let electionDisplayString = 'Unknown';
+  let electionDisplayString = 'Unknown Election';
   if (doesCardElectionHashMatchMachineElectionHash) {
     const { election } = electionDefinition;
     const electionDateFormatted = format.localeWeekdayAndDate(
       new Date(election.date)
     );
     electionDisplayString = `${election.title} — ${electionDateFormatted}`;
-  } else if (role === 'superadmin') {
-    electionDisplayString = 'N/A';
+  }
+
+  const possibleActions = new Set<SmartcardActionStatus['action']>();
+  if (
+    'passcode' in programmedUser &&
+    (role === 'superadmin' ||
+      // If the card is from a prior election, no need to display PIN resetting. Unprogramming is
+      // the only meaningful action in this case
+      doesCardElectionHashMatchMachineElectionHash)
```

Comment:
The other election may not be in the past, it may be a parallel election. Thus I think we do need ability to Reset PIN.

Reply:
Yes, this crossed my mind. That said, this change is larger in scope than meets the eye.

When we reset the PIN today, we actually just fully re-program the card, election definition and all:

https://github.com/votingworks/vxsuite/blob/a4b355a3942cca8ac2ce9465f6eae34f8ba88d61/frontends/election-manager/src/components/smartcard_modal/card_details_view.tsx#L76-L84

We haven't yet exposed an interface to just update the PIN, while leaving other contents, like a previous election def, intact. Lots of ways to go about this, none too difficult, but will require testing, etc.

Since we haven't yet built multi-election support on VxAdmin, I chose not to prioritize this for m14. That said, the current code comment is a bit misleading. Let me update the comment to shed some light on future directions. I'll also document the PIN reset updates in a follow-up task.

Reply:
https://github.com/votingworks/vxsuite/pull/2267/commits/0982d1392f52aeabfc5b13ae1f885ad0dc685757